openedx/core/djangoapps/util/waffle.py

```diff
@@ -3,16 +3,17 @@
 """
 
 
-from edx_toggles.toggles import LegacyWaffleSwitchNamespace
+from edx_toggles.toggles.__future__ import WaffleSwitch
```

Comment:
Hi, @nizarmah, as per https://github.com/edx/edx-toggles/pull/108/files I think that we should avoid the usage of __future__

```suggestion
 from edx_toggles.toggles import WaffleSwitch 
```

Reply:
Hello @morenol! Oh! Thanks for catching that 😃